app/controllers/app_groups_controller.rb

```diff
@@ -50,8 +50,10 @@ def show
     @allow_edit_app_group_name = policy(@app_group).update_app_group_name?
     @allow_delete_helm_infrastructure = policy(@app_group.helm_infrastructure).delete?
     @allow_manage_labels = policy(@app_group).update_labels?
+    @allow_manage_redact = policy(@app_group).update_redact_labels?
     @required_labels = Figaro.env.DEFAULT_REQUIRED_LABELS.split(',', -1)
     @show_log_and_cost_col = Figaro.env.SHOW_LOG_AND_COST_COL == "true"
+    @show_redact_pii = Figaro.env.SHOW_REDACT_PII == "true"
```

Comment:
with this ENV, the redact rules will applicable/shows for all app groups, right? or we will enable it for each app group only?

Reply:
correct, it will be applicable for all app groups at once. I've added a new button for redaction at app group level [here](https://github.com/BaritoLog/BaritoMarket/pull/298/commits/e22ca3639e33002111a0b54e6d397e95e5896b7e)